apps/bfd-model/bfd-model-codegen/src/main/java/gov/cms/bfd/model/codegen/RifLayoutsProcessor.java

```diff
@@ -1503,4 +1828,156 @@ private void writeDebugLogMessages() {
       throw new UncheckedIOException(e);
     }
   }
+
+  private static void createBeneficiaryMonthlyFields(
+      TypeSpec.Builder lineEntity,
+      boolean isId,
+      boolean isTransient,
+      boolean isColumnOptional,
+      String fieldName,
+      RifColumnType type,
+      Optional<Integer> columnLength,
+      Optional<Integer> columnScale) {
+    FieldSpec lineField =
+        FieldSpec.builder(
+                selectJavaFieldType(type, isColumnOptional, columnLength, columnScale),
+                fieldName,
+                Modifier.PRIVATE)
+            .addAnnotations(
+                createBeneficiaryMonthlyAnnotations(
+                    isId,
+                    isTransient,
+                    isColumnOptional,
+                    fieldName,
+                    type,
+                    columnLength,
+                    columnScale))
+            .build();
+    lineEntity.addField(lineField);
+
+    MethodSpec.Builder lineFieldGetter =
+        MethodSpec.methodBuilder(calculateGetterName(lineField))
+            .addModifiers(Modifier.PUBLIC)
+            .returns(selectJavaPropertyType(type, isColumnOptional, columnLength, columnScale));
+    addGetterStatement(isColumnOptional, lineField, lineFieldGetter);
+    lineEntity.addMethod(lineFieldGetter.build());
+
+    MethodSpec.Builder lineFieldSetter =
+        MethodSpec.methodBuilder(calculateSetterName(lineField))
+            .addModifiers(Modifier.PUBLIC)
+            .returns(void.class)
+            .addParameter(
+                selectJavaPropertyType(type, isColumnOptional, columnLength, columnScale),
+                lineField.name);
+    addSetterStatement(isColumnOptional, lineField, lineFieldSetter);
+    lineEntity.addMethod(lineFieldSetter.build());
+  }
+
+  private static List<AnnotationSpec> createBeneficiaryMonthlyAnnotations(
```

Comment:
FWIW, I'd suggest JavaDoc'ing this, as it's not entirely obvious what some of those parameters mean (I can never quite remember what scale means, for instance).